rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/parsing/parsetreetoast/misc.kt

```diff
@@ -251,7 +251,6 @@ fun RContext.toAst(conf: ToAstConfiguration = ToAstConfiguration(), source: Stri
     val fakeProcedures = getFakeProcedures(rContext = this,
```

Comment:
Considered your refactory and the fact that `fakeProcedures`, is a list of `CompilationUnit` with no statement, don't you think that the right name, rather than `getFakeProcedures` should be `getProcerurePrototypes?`